src/panel-app/apps.component.js

```diff
@@ -118,28 +162,61 @@ body.dark {
   color: #F8F8F2;
 }
 
-& .table {
-  width: 100%;
+& [role="table"] {
+  display: grid;
+  grid-row-gap: var(--table-spacing);
+  grid-template-columns: 1fr;
+  padding: var(--table-spacing);
+  width: max-content;
 }
 
-& .table td, .table th {
-  padding: 2px 8px;
+& [role="columnheader"] {
+  color: var(--gray);
+  font-size: .9rem;
+  padding-left: .25rem;
 }
 
-& .table-header {
-  color: #66D9EF;
-  text-align: left;
+& [role="row"] {
+  align-items: center;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 2fr;
+  grid-column-gap: calc(var(--table-spacing) * 2);
 }
 
+& [role="row"] [role="cell"]:nth-child(1) { grid-area: 1 / 1 / 2 / 2; }
+& [role="row"] [role="cell"]:nth-child(2) { grid-area: 1 / 2 / 2 / 3; }
+& [role="row"] [role="cell"]:nth-child(3) { grid-area: 1 / 3 / 2 / 4; }
+& [role="row"] [role="cell"]:nth-child(4), 
+& [role="row"] [role="cell"].summary-action { grid-area: 1 / 4 / 2 / 5; }
+
 & .app-status {
+  border-radius: 1rem;
+  color: #fff;
+  font-size: .75rem;
+  padding: .25rem .5rem .125rem;
+  text-shadow: 0px 2px 4px rgba(0,0,0,.15);
   text-transform: capitalize;
 }
 
 & .app-mounted {
-  color: #A6E22E;
+  background-color: var(--green);
 }
 
 & .app-not-mounted {
-  color: #F92672;
+  background-color: var(--pink);
+}
+
+& .import-override {
+  border: 1.5px solid lightgrey;
+  border-radius: 3px;
+  box-sizing: border-box;
+  font-size: .75rem;
+  padding: .2rem;
+  transition: all .15s ease-in-out;
```

Comment:
SUPER nit: I personally am not a fan of transition all; I like listing out exactly what I expect to transition. it just makes it more clear what you intend to happen and you avoid weird transitions that happen accidentally like if a screen rotates/resizes and now you're transitioning the width by accident or something.

not a big deal at all though.

Reply:
That's fine with me.